### ISSUE TITLE
Files have wrong permissions

### ISSUE THREAD

**Issue by kseistrup** [chevdor/nrs-dividends_scanner#1]
When cloning the repository to a unix like operating system, e.g. Linux, the files come up as executable.  Please change permissions from 0755 (`-rwxr-xr-x`) to 0644 (`-rw-r--r--`).

Here's a diff:

``` diff
diff --git a/Readme.adoc b/Readme.adoc
old mode 100755
new mode 100644
diff --git a/css/dividends_scanner.css b/css/dividends_scanner.css
old mode 100755
new mode 100644
diff --git a/html/modals/dividends_scanner.html b/html/modals/dividends_scanner.html
old mode 100755
new mode 100644
diff --git a/html/pages/dividends_scanner.html b/html/pages/dividends_scanner.html
old mode 100755
new mode 100644
diff --git a/js/nrs.dividends_scanner.js b/js/nrs.dividends_scanner.js
old mode 100755
new mode 100644
diff --git a/manifest.json b/manifest.json
old mode 100755
new mode 100644
```

**Comment by chevdor** [comment]
Hello @kseistrup 
Thanks for reporting this issue. 

The extra rights were indeed not required and this is now fixed. It will be included in the next release.

I will not make a new release just for this change and it sounds like you could make the change on your side. 

I hope you like the plugin and find it useful. Feel free to report other issues you may run into.

**Comment by kseistrup** [comment]
Thanks.

(I've already made the changes on my side, but I wanted them to propagate to the origin.)

**Comment by chevdor** [comment]
Actually, since I just pushed a new version, your changes are available in v0.4.0.0
Thanks again for sharing.
